src/gui/gridview.cpp

```diff
@@ -100,7 +100,7 @@ void GridView::paintFrame(QPainter& painter) const
     }
 }
 
-Node GridView::findNode(const QPointF& pos) const
+Node GridView::findNode(const QPointF& pos)
```

Comment:
Because we may want to move the node by changing it's coordinates, so ``const`` would throw a compilation error. ``selectNode`` wouldn't work, since we may want to move the node without adding it to our current selection. Maybe it would make more sense to rename ``findNode`` to ``getNode``?

Reply:
nope, any "find" or "get" function should be const
it seems that you are mixing two different concepts here: 1) const function, 2) const return

any getter/finder function should be const because it should never ever change a member variable

could you revert this change please? (i.e., make this function const)